augur/utils.py

```diff
@@ -550,21 +555,48 @@ def read_strains(*files, comment_char="#"):
     Parameters
     ----------
     files : iterable of str
-        one or more names of text files with one strain name per line
+        one or more names of text files with one entry per line
 
     Returns
     -------
     set :
-        strain names from the given input files
+        lines from the given input files
 
     """
-    strains = set()
+    entries = list()
     for input_file in files:
         with open_file(input_file, 'r') as ifile:
             for line in ifile:
                 # Allow comments anywhere in a given line.
-                strain_name = line.split(comment_char)[0].strip()
-                if len(strain_name) > 0:
-                    strains.add(strain_name)
+                entry = line.split(comment_char)[0].strip()
+                if len(entry) > 0:
+                    entries.append(entry)
+
+    return entries
+
+
+def get_genes(input):
```

Comment:
Can this function be called `interpret_genes_argument` or `parse_genes_argument` or some such? I see util functions as ones which may be imported in multiple places and `get_genes` seems overly generic to me.

Reply:
Renamed to `parse_genes_argument` in [force-push](https://github.com/nextstrain/augur/compare/f59f1808e9849eb8b0cab1e20d4654cf91eced4c..a60f1a63f7140e738d4e29ae67bd9c72f4c88268).

I considered specifying it as `type` for argparse directly (like [`numeric_date_type`](https://github.com/nextstrain/augur/blob/a60f1a63f7140e738d4e29ae67bd9c72f4c88268/augur/dates/__init__.py#L68)), but I don't think it works well with `nargs='+'`.